source/unified-test-format/unified-test-format.rst

```diff
@@ -1332,6 +1333,12 @@ that the field is set and is a non-empty BSON ObjectId (i.e. all bytes of the
 ObjectId are not 0). If false, test runners MUST assert that the field is not
 set or is an empty BSON ObjectId.
 
+closeInUseConnections
+
+This field is an optional boolean that specifies whether or not the
+``closeInUseConnections`` field should be asserted. If field is set, test runners MUST assert
+that the field is set and has appropriate value. If not set, test runners MUST ignore asserting.
+
```

Comment:
IIRC, I previously suggested removing this entirely when inlining `closeInUseConnections` under `poolClearedEvent`. This paragraph adds nothing to what is already discussed above. If this option is ever generalized and applied to multiple events that'd be a reason to move it here.

Additionally, this no longer has heading syntax and would render as a paragraph under `hasServerId`.

```suggestion
```